sn_transfers/src/wallet/utxo.rs

```diff
@@ -22,6 +24,38 @@ pub struct Transfer {
 }
 
 impl Transfer {
+    /// Creates Transfers from the given dbcs
+    /// Grouping DBCs by recipient into different transfers
+    /// This Transfer can be sent safely to the recipients as all data in it is encrypted
+    /// The recipients can then decrypt the data and use it to verify and reconstruct the DBCs
+    pub fn transfers_from_dbcs(dbcs: Vec<Dbc>) -> Result<Vec<Transfer>> {
```

Comment:
This function to create Transfer from DBCs, **can be done offline**, and sent to the recipient.